functests/1_performance/performance.go

```diff
@@ -253,17 +253,17 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 				ksoftirq_prio, err := strconv.Atoi(match[1])
 				Expect(err).ToNot(HaveOccurred())
 
-				rcuc_pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "rcuc", "-n"}, &node)
+				rcu_pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "rcu[c,b]", "-n"}, &node)
```

Comment:
it would be nice to document when we should expect kernel threads named `rcuc` and when `rcub`. What's the difference?

Reply:
I added two notes according to these articles :
https://www.kernel.org/doc/html/latest/admin-guide/kernel-per-CPU-kthreads.html 
https://lwn.net/Articles/220677/